### PR TITLE
lang/updater: small fix grammar of English for updater.

### PR DESCRIPTION
**Pull request by Zangetsu38**

### CODE REVIEW

File: vita3k/lang/include/lang/state.h [hunk 1]
```diff
@@ -398,14 +398,14 @@ struct LangState {
...
-        { "new_version_available", "A new version of the Vita3K is available." },
+        { "new_version_available", "A new version of Vita3K is available." },
         { "back", "Back" },
         { "downloading", "Downloading...\nAfter the download is complete, Vita3K will restart automatically and then install the new Vita3K." },
         { "not_complete_update", "Could not complete the update." },
         { "next", "Next" },
         { "update_vita3k", "Do you want to update Vita3K" },
-        { "later_version_already_installed", "The later version of the Vita3K software is already installed." },
-        { "latest_version_already_installed", "The latest version of the Vita3K is already installed." },
+        { "later_version_already_installed", "The later version of Vita3K is already installed." },
```
Comment (EXtremeExploit) [inline vita3k/lang/include/lang/state.h:407]: would make more sense if it said "A later version" instead of "The later version" :thinking:

Comment (Zangetsu38) [reply vita3k/lang/include/lang/state.h:407]: No is official string

File: lang/en.xml [hunk 2]
```diff
@@ -475,15 +475,15 @@ Are you sure you want to continue?</user_delete_warn>
...
+ 		<new_version_available>A new version of Vita3K is available.</new_version_available>
 		<back>Back</back>
 		<downloading>Downloading...
 After the download is complete, Vita3K will restart automatically and then install the new Vita3K.</downloading>
 		<not_complete_update>Could not complete the update.</not_complete_update>
 		<next>Next</next>
 		<update_vita3k>Do you want to update Vita3K?</update_vita3k>
-		<later_version_already_installed>The later version of the Vita3K software is already installed.</later_version_already_installed>
-		<latest_version_already_installed>The latest version of the Vita3K is already installed.</latest_version_already_installed>
+		<later_version_already_installed>The later version of Vita3K software is already installed.</later_version_already_installed>
```
Comment (EXtremeExploit) [inline lang/en.xml:485]: deleted software on lang state but not on xml languages :thinking:

Comment (Zangetsu38) [reply lang/en.xml:485]: OUPSY __

**Review by EXtremeExploit — APPROVED** [review]

**Pull request merged**